src/matches.js

```diff
@@ -17,16 +13,12 @@ function fuzzyMatches(
   }
 }
 
-function matches(
-  textToMatch,
-  node,
-  matcher,
-  {collapseWhitespace = true, trim = true} = {},
-) {
+function matches(textToMatch, node, matcher, normalizer) {
```

Comment:
Is there a way we can add this option without making a breaking change to the matches APIs?

Reply:
I guess we can ignore this if removing the matchers as exports per https://github.com/kentcdodds/dom-testing-library/pull/172#discussion_r240465361